samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -29,26 +18,39 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf52833dk_nrf52833
 
-The sample also requires an antenna matrix when operating in Angle of Departure mode.
+The sample also requires an antenna matrix when operating in angle of departure mode.
 It can be a Nordic Semiconductor design 12 patch antenna matrix, or any other antenna matrix.
 
+Overview
+********
+
+The direction finding connectionless beacon sample application uses Constant Tone Extension (CTE), that is transmitted with periodic advertising PDUs.
+It uses the Constant Tone Extension (CTE), that is transmitted with periodic advertising PDUs.
```

Comment:
I think we can remove the second sentence now.

Reply:
Sentence removed.